content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -187,7 +187,7 @@ For a reference to old feature gates that are removed, please refer to
 | `SELinuxMountReadWriteOncePod` | `true` | Beta | 1.27 | |
 | `SecurityContextDeny` | `false` | Alpha | 1.27 | |
 | `ServiceNodePortStaticSubrange` | `false` | Alpha | 1.27 | 1.27 |
-| `ServiceNodePortStaticSubrange` | `true` | Beta | 1.28 | |
+| `ServiceNodePortStaticSubrange` | `true` | Beta | 1.28 | 1.28 |
```

Comment:
When a gate is GA'ed, the gate should be mentioned only in the second table.

Reply:
We are supposed to remove line 189 and 190, that is what we call "moving" rather than "copying", right?